scripts/get_sys_observability_data.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+echo "Input requested parameters for fetching observability data."
+echo "If empty value, press ENTER"
+
+echo "Input clustername"
+read clustername
+
+echo "Input containername"
+read containername
+
+echo "Input namespace"
+read namespace
+
+echo "Input labels"
+read labels
+
+DATA='{"clusterName": "'$clustername'", "namespace":"'$namespace'", "containerName":"'$containername'", "labels":"'$labels'"}'
```

Comment:
Is it necessary that all the parameters are provided?

If no clustername is provided .. then we should be using all clusters ... Similarly for other fields.

Reply:
Yes Rahul. If we did not provide cluster name, all clusters will be read. If any vallue is ignored then * is used in query